_data/advisory.yml

```
@@ -0,0 +1,76 @@
+- group: steering
```

```suggestion
- group: advisory
```

? since steering is another thing